src/StorageSync/StorageSync/debug.ps1

```diff
@@ -0,0 +1,226 @@
+param(
```

Comment:
It looks liek this script is not part of test automation - please do not check-in scripts that are not used in build/tests